inputs/orchestrator_inner:4.json

```diff
@@ -163,6 +163,9 @@
     },
     {
       "name": "remove_built_image"
+    },
+    {
+      "name": "remove_worker_metadata"
```

Comment:
Since removing the configmap after use is not essential for a successful build, I would suggest adding:
```
"required": False
```

so that osbs-client can still create successful builds even when using a builder image that does not have the remove_worker_metadata plugin present.

Reply:
added